bar/90_contents/src/login.html

```diff
@@ -225,7 +243,7 @@
                         </br>
                         <label style="display: table;margin: 0 auto;margin-top: -14px">of Google Chrome.</label>
                     </div>
-                    <form name="login" id="loginForm" style="margin-left: 106px;">
+                    <form name="login" id="loginForm" style="margin-top:90px;margin-left: 106px;">
```

Comment:
Add CSS instead of modifying style so that vertical alignment can be done automatically.

        #loginForm {
            margin-left: 106px;
        }

        #loginForm.localCellManager {
            margin-top:90px;
        }